enclave_build/src/lib.rs

```diff
@@ -73,6 +83,33 @@ impl<'a> Docker2Eif<'a> {
             return Err(Docker2EifError::ArtifactsPrefixError);
         }
 
+        let meta_file = match meta_file_path {
+            Some(meta) => {
+                if !Path::new(&meta).is_file() {
+                    return Err(Docker2EifError::MetadataPathError);
+                } else {
+                    Some(meta.clone())
+                }
+            }
+            None => None,
+        };
+
+        let uri_split: Vec<&str> = docker_image.split(':').collect();
+        let repo = uri_split[0].to_string();
```

Comment:
What if ```uri_split``` has no elements? You should check this and return an error here.

Reply:
Added check for `is_empty()`